File: 3-Strings/3141-Duvida-Etaria/3141.py
name = input()
now = input().split('/')
birth = input().split('/')
dates = {}
cont = 0
for _ in now:
    dates.update({cont : int(_)})
    cont += 1
for _ in birth:
    dates.update({cont : int(_)})
    cont += 1
if dates[0] == dates[3] and dates[1] == dates[4]:
    print ("Feliz aniversario!")
if dates[1] > dates[4]:
    print ("Voce tem", (dates[2] - dates[5]), "anos {}." .format(name))
elif dates[1] < dates[4]:
    print ("Voce tem", (dates[2] - dates[5]) - 1, "anos {}." .format(name))
elif dates[1] == dates[4] and dates[0] < dates[3]:
    print ("Voce tem", (dates[2] - dates[5]) - 1, "anos {}." .format(name))
elif dates[1] == dates[4] and dates[0] == dates[3]:
    print ("Voce tem", (dates[2] - dates[5]), "anos {}." .format(name))

File: 3-Strings/3141-Duvida-Etaria/3141.cpp
#include <cstdio>
#include <cstring>

int main() {
    char name[51];
	int date[6];
	fgets(name, 51, stdin);
	name[strcspn(name, "\n")] = '\0';
	scanf ("%d/%d/%d\n%d/%d/%d", &date[0], &date[1], &date[2], &date[3], &date[4], &date[5]);
	if (date[0] == date[3] && date[1] == date[4]) {
	    printf ("Feliz aniversario!\n");
	}
	if (date[1] == date[4] && date[0] < date[3] || date[1] < date[4]) {
	    printf ("Voce tem %d anos %s.\n", date[2] - date[5] - 1, name);
	}
	else {
	    printf ("Voce tem %d anos %s.\n", date[2] - date[5], name);
	}
	
	
	return 0;
}